lang/libcxx/Portfile

```diff
@@ -89,11 +90,18 @@ if {${os.major} < 11 || [variant_isset replacemnt_libcxx]} {
 
         # only selected clang versions support emulated_tls, and the emulated_tls variant
         # needs to be enabled to build libcxx with emulated_tls support
-        foreach ver {5.0 6.0 7.0 8.0 9.0} {
-            if {[file exists ${prefix}/bin/clang-mp-${ver}]} {
-                if {[active_variants clang-${ver} emulated_tls]} {
-                  default_variants-append +emulated_tls
-                  compiler.whitelist-prepend macports-clang-${ver}
+        foreach clang_c ${compiler.fallback} {
+            if {[string match macports-clang-* $clang_c]} {
+                # Extract version number from macports clang compiler name
+                set clang_v [lindex [split $clang_c -] 2]
+                if { ${clang_v} >= 5.0 } {
+                    if {[file exists ${prefix}/bin/clang-mp-${clang_v}]} {
+                        if {[active_variants clang-${clang_v} emulated_tls]} {
+                            ui_debug "Adding ${clang_c} to compiler whitelist"
+                            default_variants-append +emulated_tls
+                            compiler.whitelist-prepend ${clang_c}
```

Comment:
Why reverse the order? For that matter, why add more than one macports-clang version to the whitelist? The first one will always be used.

Reply:
My changes do not alter what the original portfile does, and it was strictly my intention to do that.